config/create.sql

```diff
@@ -0,0 +1,147 @@
+/* SQL commands to create the tables used by this version of Rose City Resource */
+
+/*
+    NOTE:
```

Comment:
Thank you for this context.

Reply:
We should probably add this info to the wiki/readme.

Reply:
Done